src/main/java/group1/mips_simulator/components/memParts/Storage.java

```diff
@@ -0,0 +1,9 @@
+package group1.mips_simulator.components.memParts;
+import group1.mips_simulator.components.Value;
+
+public class Storage {
+    public Value[] data;
+    public Storage(int size){
+        data = new Value[size];
```

Comment:
So, all the values in the in the `data` array are initialized to `null`.  Please add a loop here that creates a `new Value(0)` object, or if you want a `new Value(memLocation)` or whatever. 

But as written, I don't think this will work.

Reply:
you're right, i forgot that it won't just initialize empty spots to zero lol, i will add this in now

Reply:
oh thats weird, the commit didnt...include my change?

Reply:
but it did appear in the commited version. weird